### DISCUSSION TITLE
Is there a way to make a part add 2 objects in the metadata?

### DISCUSSION THREAD

**Discussion by VitorAugustoCunha** [nftchef/art-engine#286] [Help + Q&A]
I have a part of a hair and a hat but the hair passes the hat, I would like to know if there is a way I can make a specific design for that hat with the hair, it would just be an image containing 2 parts of the metadata example:

Hair1 + Hat1 = HAIR1andHAT1

**Comment by nftchef** [comment]
There is no way to do this at this time. 
Two or more images can be combined into one "layer" folder and stacked, but both traits would be present separately in the metadata, for example.

```
hat: "hat 1",
hair: "hair 1"
```

depending on the need for this kind of solution, altering the metadata after image generation (custom script) would be the easiest approach.

**Comment by null-prophet** [comment]
I have adapted the code here locally to do something like this. 

Adding in a function that reads the DNA in and then based on regex/filters add new traits to the metadata. It's very specific to my project/tests but its adaptable and adds a little bit of time onto the processing but makes it possible to add in game like values specific to a certain DNA element that was picked.
